### ISSUE TITLE
Docker hub image jdkato/vale not built for Apple M1 chips

### ISSUE THREAD

**Issue by robolyst** [jdkato/txtlint#532]
### Problem

I'm on a MacBook Pro M1 machine. When I execute the command:
```
docker run jdkato/vale
```

I get
```
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
```

### Potential solution
I'm not sure where the build and push config is for the image. But, I believe we need to add `linux/arm64` as a build platform to `docker build`?

For example, if there's a github action to build and push the image to Docker hub you would go:
```yaml
- name: Build and push Docker image
  uses: docker/build-push-action@ad44023a93711e3deb337508980b4b5e9bcdc5dc
  with:
    context: .
    platforms: linux/amd64,linux/arm64
    push: true
```

**Comment by jdkato** [comment]
This should be fixed now.

**Comment by robolyst** [comment]
Fantastic, thank you 🙏🏻
